fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/executor/Analyzer.java

```diff
@@ -125,6 +127,8 @@ private static List<RewriteJob> buildAnalyzeJobs(Optional<CustomTableResolver> c
             topDown(new EliminateGroupByConstant()),
             topDown(new NormalizeAggregate()),
             bottomUp(new JoinCommute()),
+            topDown(new CalcFoundRows()),
+            topDown(new FoundRows()),
```

Comment:
we call setFoundRowsPlan in CalcFoundRows rule. and getFoundRowsPlan in FoundRows, so we need apply FoundRows in the head of CalcFoundRows

Reply:
done